scrapers/freshmaker.py

```diff
@@ -86,12 +86,11 @@ def query_api_and_update_neo4j(self):
 
                 for build_dict in fm_event['builds']:
                     # To handle a faulty container build in Freshmaker
-                    if not build_dict['build_id'] or int(build_dict['build_id']) < 0:
+                    if int(build_dict['build_id']) < 0:
```

Comment:
Could you please explain this change? This code was protecting against `build_dict['build_id']` was `None`. You can see that the model doesn't have `nullable=False` set:
https://pagure.io/freshmaker/blob/master/f/freshmaker/models.py#_501

Reply:
Yeah I see. But the request is exactly to show in Estuary those builds... Example: https://pipeline.engineering.redhat.com/freshmakerevent/13277

Can we change the constrict in the model?

Reply:
I think changing this line to the following would alleviate my concerns:
```python
if build_dict['build_id'] and int(build_dict['build_id']) < 0:
```

Reply:
Sounds good! Thank you, Matt.
I've addressed the comment.